addons/website_forum/models/forum_post.py

```diff
@@ -868,3 +868,34 @@ def _search_render_results(self, fetch_fields, mapping, icon, limit):
             if with_date:
                 data['date'] = self.env['ir.qweb.field.date'].record_to_html(post, 'write_date', {})
         return results_data
+
+    def _get_related_posts(self, limit=5):
+        if not self.tag_ids:
+            return None
+
+        self.flush_model()
+        self.env["forum.tag"].flush_model()
```

Comment:
with the current state of the code it's indeed useless (we don't write in the controller), but if in the future we use that function somewhere else, it might be needed (so it was more to follow good practice, but I'm ok to remove it)